packages/core/src/App/Containers/app-notification-messages.jsx

```diff
@@ -107,38 +107,39 @@ const AppNotificationMessages = ({
         const is_not_marked_notification = !marked_notifications.includes(message.key);
         const is_non_hidden_notification = isMobile()
             ? [
-                  'unwelcome',
-                  'contract_sold',
-                  'dp2p',
-                  'install_pwa',
-                  'tnc',
-                  'need_fa',
+                  'authenticate',
                   'deriv_go',
+                  'document_needs_action',
+                  'dp2p',
                   'close_mx_mlt_account',
-                  'trustpilot',
                   'close_uk_account',
-                  'p2p_daily_limit_increase',
-                  'document_needs_action',
+                  'contract_sold',
+                  'has_changed_two_fa',
                   'identity',
+                  'install_pwa',
+                  'need_fa',
                   'poi_name_mismatch',
+                  'poa_address_mismatch_failure',
+                  'poa_address_mismatch_success',
+                  'poa_address_mismatch_warning',
+                  'poa_expired',
+                  'poa_failed',
+                  'poa_rejected_for_mt5',
+                  'poa_verified',
                   'poi_expired',
                   'poi_failed',
                   'poi_verified',
-                  'poa_expired',
-                  'resticted_mt5_with_pending_poa',
-                  'poa_verified',
-                  'poa_failed',
+                  'p2p_daily_limit_increase',
                   'resticted_mt5_with_failed_poa',
-                  'poa_rejected_for_mt5',
-                  'poa_address_mismatch_warning',
-                  'poa_address_mismatch_success',
-                  'poa_address_mismatch_failure',
-                  'svg_needs_poi_poa',
+                  'resticted_mt5_with_pending_poa',
                   'svg_needs_poa',
-                  'has_changed_two_fa',
                   'svg_needs_poi',
+                  'svg_needs_poi_poa',
                   'svg_poi_expired',
                   'switched_to_real',
+                  'tnc',
+                  'trustpilot',
+                  'unwelcome',
               ].includes(message.key) || message.type === 'p2p_completed_order'
             : true;
 
```

Comment:
do we need this change as part of this PR?

Reply:
It looks like a reorder of the list elements